src/docs/product/discover-queries/index.mdx

```diff
@@ -90,9 +92,7 @@ Queries can also be added to <SandboxLink scenario="dashboards" projectSlug="rea
 
 ### Set as Default
 
-<Include name="early-adopter-note.mdx" />
-
-If your organization is part of the Early Adopter program, when you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query.
+When you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query.
 
 You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded.
```

Comment:
Wondering if you can set other pre-built queries as defaults? Should we be mentioning this?
```suggestion
You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded. The "Set as Default" button is also available on saved queries, which you can find by clicking the "Saved Queries" button.
```

Reply:
Updated my suggestion

Reply:
Nevermind